rotkehlchen/tests/unit/decoders/kyber.py

```diff
@@ -0,0 +1,223 @@
x1f\x96(\xc0G\xaf\x1f\x8b\x19\x1c\xbf\xfb\xb6\xb8\xb9\x94Z\xdbP,\xfb:\xf1RU/"', 367, b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00|fU\x0c\x9cs\x0bo\xddL\x03\xbc.s\xc5F,_z\xcc', 2), (b'\xe8\t(\xd5\xe2\x1f\x96(\xc0G\xaf\x1f\x8b\x19\x1c\xbf\xfb\xb6\xb8\xb9\x94Z\xdbP,\xfb:\xf1RU/"', 368, b'\xea\x94\x158[\xae\x08\xfe\x9fm\xc4W\xb0%w\x16g\x90\xcd\xe8;\xb1\x8c\xc3@\xaa\xc6\xcb\x81\xb8$\xde', 0), (b'\xe8\t(\xd5\xe2\x1f\x96(\xc0G\xaf\x1f\x8b\x19\x1c\xbf\xfb\xb6\xb8\xb9\x94Z\xdbP,\xfb:\xf1RU/"', 369, b'\xdd\xf2R\xad\x1b\xe2\xc8\x9bi\xc2\xb0h\xfc7\x8d\xaa\x95+\xa7\xf1c\xc4\xa1\x16(\xf5ZM\xf5#\xb3\xef', 0), (b'\xe8\t(\xd5\xe2\x1f\x96(\xc0G\xaf\x1f\x8b\x19\x1c\xbf\xfb\xb6\xb8\xb9\x94Z\xdbP,\xfb:\xf1RU/"', 369, b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00|fU\x0c\x9cs\x0bo\xddL\x03\xbc.s\xc5F,_z\xcc', 1), (b'\xe8\t(\xd5\xe2\x1f\x96(\xc0G\xaf\x1f\x8b\x19\x1c\xbf\xfb\xb6\xb8\xb9\x94Z\xdbP,\xfb:\xf1RU/"', 369, b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00S@\xf6\xfa\xff\x9b\xf5_f\xc1m\xb6\xbf\x9e\x02\r\x98\x7f\x87\xd0', 2), (b'\xe8\t(\xd5\xe2\x1f\x96(\xc0G\xaf\x1f\x8b\x19\x1c\xbf\xfb\xb6\xb8\xb9\x94Z\xdbP,\xfb:\xf1RU/"', 370, b'\xc2\x07\xa6<\x18\xc4\x07\x0c\xe1\xe3>_\xcc\x02\xef\xb0\x9a\xc9\x84\xca\xa6\xa2\x04n+\x1d(\x11r8F\xf1', 0), (b'\xe8\t(\xd5\xe2\x1f\x96(\xc0G\xaf\x1f\x8b\x19\x1c\xbf\xfb\xb6\xb8\xb9\x94Z\xdbP,\xfb:\xf1RU/"', 370, b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xee\xee\xee\xee\xee\xee\xee\xee\xee\xee\xee\xee\xee\xee\xee\xee\xee\xee\xee\xee', 1), (b'\xe8\t(\xd5\xe2\x1f\x96(\xc0G\xaf\x1f\x8b\x19\x1c\xbf\xfb\xb6\xb8\xb9\x94Z\xdbP,\xfb:\xf1RU/"', 370, b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00|fU\x0c\x9cs\x0bo\xddL\x03\xbc.s\xc5F,_z\xcc', 2), (b'\xe8\t(\xd5\xe2\x1f\x96(\xc0G\xaf\x1f\x8b\x19\x1c\xbf\xfb\xb6\xb8\xb9\x94Z\xdbP,\xfb:\xf1RU/"', 370, b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xdec\xae\xf6\x03\x07eT\x05\x83]\xa7K\xa0,\xe4\xdb\x1aB\xfb', 3), (b'\xe8\t(\xd5\xe2\x1f\x96(\xc0G\xaf\x1f\x8b\x19\x1c\xbf\xfb\xb6\xb8\xb9\x94Z\xdbP,\xfb:\xf1RU/"', 371, b'0\xbb\xea`:{6\x85\x8f\xe5\xe3\xeck\xa5\xffY\xdd\xe09\xd0! \xd7X\xea\xcf\xae\xd0\x15 W}', 0), (b'\xe8\t(\xd5\xe2\x1f\x96(\xc0G\xaf\x1f\x8b\x19\x1c\xbf\xfb\xb6\xb8\xb9\x94Z\xdbP,\xfb:\xf1RU/"', 371, b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xa0\xb8i\x91\xc6!\x8b6\xc1\xd1\x9dJ.\x9e\xb0\xce6\x06\xebH', 1), (b'\xe8\t(\xd5\xe2\x1f\x96(\xc0G\xaf\x1f\x8b\x19\x1c\xbf\xfb\xb6\xb8\xb9\x94Z\xdbP,\xfb:\xf1RU/"', 371, b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xd53\xa9It\x0b\xb30m\x11\x9c\xc7w\xfa\x90\x0b\xa04\xcdR', 2), (b'\xe8\t(\xd5\xe2\x1f\x96(\xc0G\xaf\x1f\x8b\x19\x1c\xbf\xfb\xb6\xb8\xb9\x94Z\xdbP,\xfb:\xf1RU/"', 372, b'\xf7$\xb4\xdff\x17G6\x12\xb5=\x7f\x88\xec\xc6\xea\x980t\xb3\t`\xa0I\xfc\xd0e\x7f\xfe\x80\x80\x83', 0), (b'\xe8\t(\xd5\xe2\x1f\x96(\xc0G\xaf\x1f\x8b\x19\x1c\xbf\xfb\xb6\xb8\xb9\x94Z\xdbP,\xfb:\xf1RU/"', 372, b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00S@\xf6\xfa\xff\x9b\xf5_f\xc1m\xb6\xbf\x9e\x02\r\x98\x7f\x87\xd0', 1)],  # noqa: E501
```

Comment:
Here and before you seem to be putting a hell of a lot of log events for the transaction. Save space and only put the relevant ones that the test is going to decode. This takes too much space as you currently have it.

Reply:
I believed it was needed to correctly decode the send/receive events as when I removed part of them I was loosing this events. I've removed most of the data and only left the methods that do the swaps and the send of the tokens